DH_Engine/Classes/DHArtillerySpottingScope.uc

```diff
@@ -9,21 +9,22 @@ class DHArtillerySpottingScope extends ROVehicle
 // Range table
 struct SRangeTableRecord
 {
-    var float Mils;     // Pitch, in mils.
-    var float Range;    // Range, in meters.
-    var float TTI;      // Time-to-impact in seconds
+    var int Degrees;                                    // Pitch, in degrees.
```

Comment:
Just use `Pitch` or `Elevation` as a name or something. It's confusing since the values can be in degrees or mils.